apps/definition_accept/lib/accept.ex

```diff
@@ -3,15 +3,14 @@ defmodule Accept do
   Accept defines the structure of an accept operation
   as performed by the receive service. Accept provides
   a reference to the dataset identifier for the data to
-  be received, an unique identifier for the accept operation
-  and human-readable name, as well as a destination
-  to write the data to.
+  be received, a human-readable name, and a destination
+  to write the data to. Upon validation a UUID is
```

Comment:
Good catch updating the documentation!

Reply:
The description of the destination still needs work, but thanks.